rpms/SPECS/wazuh-manager.spec

```diff
@@ -332,7 +332,7 @@ fi
 
 if [ -f /etc/os-release ]; then
   source /etc/os-release
-  if [ "${NAME}" = "Red Hat Enterprise Linux" ] && [ "$((${VERSION_ID:0:1}))" -ge 9 ]; then
+  if ( [ "${NAME}" = "Red Hat Enterprise Linux" ] || [ "${NAME}" = "Rocky Linux" ] || [ "${NAME}" = "AlmaLinux" ] ) && [ "$((${VERSION_ID:0:1}))" -ge 9 ]; then
```

Comment:
This should also cover the corespondent versions of CentOS and Fedora, if possible we should use a generic command to determine these systems.

Reply:
The generic command would be useful but the only common variable in `/etc/os-release` is `ID_LIKE`, which function is this type of things. The problem is it is also the same for Oracle Linux 9, which doesn't have this problem.

Reply:
CentOS 9 Stream and Fedora 34 added in https://github.com/wazuh/wazuh-packages/pull/2053/commits/08bd1075fac0413d8095fa31404ff6a6b0283032